config/initializers/job_configurations.rb

```diff
@@ -163,6 +164,10 @@
         cron: cron_24h,
         args: -> { [Time.zone.yesterday] },
       },
+      usps_auth_token_refresh: {
+        class: 'UspsAuthTokenRefreshJob',
+        cron: cron_12m,
+      },
```

Comment:
I recommend adding a flag here so we can disable the job. We may also be able to reduce some log noise and overhead by defaulting the job to be disabled in local dev environments.

Reply:
Sounds good, added in [156bcd1](https://github.com/18F/identity-idp/pull/8721/commits/156bcd10a50d0e35537509ef34143860c1eb16bc)!